backend/logistica_servicio/gestor_entregas/infraestructura/modelos.py

```
@@ -0,0 +1,62 @@
+from sqlalchemy import Column, Integer, String, Float, Date, Time, ForeignKey
+from sqlalchemy import Column, Integer, String, Float, Date, Time, ForeignKey
```

Ruff (0.8.2)

Redefinition of unused `Column` from line 1

Remove definition: `Column`

(F811)